h2o-algos/src/main/java/hex/schemas/GAMV3.java

```diff
@@ -0,0 +1,241 @@
+import water.api.schemas3.StringPairV3;
+
+public class GAMV3 extends ModelBuilderSchema<GAM, GAMV3, GAMV3.GAMParametersV3> {
+  public static final class GAMParametersV3 extends ModelParametersSchemaV3<GAMModel.GAMParameters, GAMParametersV3> {
+    public static final String[] fields = new String[] {
+            "model_id",
+            "training_frame",
+            "validation_frame",
+            "nfolds",
+            "seed",
+            "keep_cross_validation_models",
+            "keep_cross_validation_predictions",
+            "keep_cross_validation_fold_assignment",
+            "fold_assignment",
+            "fold_column",
+            "response_column",
+            "ignored_columns",
+            "ignore_const_cols",
+            "score_each_iteration",
+            "offset_column",
+            "weights_column",
+            "family",
+            "tweedie_variance_power",
+            "tweedie_link_power",
+            "theta", // equals to 1/r and should be > 0 and <=1, used by negative binomial
+            "solver",
+            "alpha",
+            "lambda",
+            "lambda_search",
+            "early_stopping",
+            "nlambdas",
+            "standardize",
+            "missing_values_handling",
+            "plug_values",
+            "compute_p_values",
+            "remove_collinear_columns",
+            "intercept",
+            "non_negative",
+            "max_iterations",
+            "objective_epsilon",
+            "beta_epsilon",
+            "gradient_epsilon",
+            "link",
+            "prior",
+            "lambda_min_ratio",
+            "beta_constraints",
+            "max_active_predictors",
+            "interactions",
+            "interaction_pairs",
+            "obj_reg",
+            "export_checkpoints_dir",
+            // dead unused args forced here by backwards compatibility, remove in V4
+            "balance_classes",
+            "class_sampling_factors",
+            "max_after_balance_size",
+            "max_confusion_matrix_size",
+            "max_hit_ratio_k",
+            "max_runtime_secs",
+            "custom_metric_func",
+            "k",  // array: number of knots for each predictor
+            "knots_keys", // string array storing frame keys that contains knot location
+            "gam_x",  // array: predictor column names array
+            "bs", // array, name of basis functions used
+            "scale", // array, smoothing parameter for GAM,
+            "save_gam_cols"
+    };
+
+    @API(help = "Seed for pseudo random number generator (if applicable)", gridable = true)
+    public long seed;
+
+    // Input fields
+    @API(help = "Family. Use binomial for classification with logistic regression, others are for regression problems.", values = {"gaussian", "binomial","quasibinomial","ordinal", "multinomial", "poisson", "gamma", "tweedie", "negativebinomial"}, level = Level.critical)
+    // took tweedie out since it's not reliable
```

Comment:
Should "tweedie" be removed from the help? Also, then, should tweedie_variance_power and tweedie_link_power be removed?

Reply:
@wendycwong If this still applies, I'll remove it.

Reply:
@angela0xdata @Pscheidl 
I would like GAM to support the same families as GLM.  Since GLM uses Tweedie, GAM should too.  Please do not remove it.

Reply:
Can we remove the "//took tweedie out since it's not reliable" comment?